pkg/controller/storagecluster/storage_nodes_count.yaml

```diff
@@ -0,0 +1,1501 @@
+rows:
```

Comment:
why do we have duplicate yaml file in 2 different folders? we can move this into operator/deploy folder? that's where it should be.